packages/api/src/types.ts

```diff
@@ -1,13 +1,46 @@
 import { Static, Type, TSchema } from '@sinclair/typebox'
 export { Db, Collection, ObjectId, WithId } from 'mongodb'
 
-const Nullable = <T extends TSchema>(type: T) => Type.Union([type, Type.Null()])
+export const Nullable = <T extends TSchema>(type: T) =>
+  Type.Union([type, Type.Null()])
 
 export const ClaimPlayerParams = Type.Object({
   key: Type.String(),
 })
 export type ClaimPlayerParams = Static<typeof ClaimPlayerParams>
 
+// Socials
+
+export const Socials = Nullable(
+  Type.Object({
+    twitter: Type.Optional(Type.String()),
+    discord: Type.Optional(Type.String()),
+    telegram: Type.Optional(Type.String()),
+    name: Type.Optional(Type.String()),
+    company: Type.Optional(Type.String()),
+    share: Type.Optional(Type.Boolean()),
```

Comment:
Should `share` be inside `Socials`?  When we store the socials we are also saving this fields which has no relation to the Player storing other Player socials